app/bundles/EmailBundle/Controller/PublicController.php

```diff
@@ -422,8 +421,8 @@ public function mailerCallbackAction($transport)
         $transportParam   = $this->get('mautic.helper.core_parameters')->getParameter(('mailer_transport'));
         $currentTransport = $this->get('swiftmailer.mailer.transport.'.$transportParam);
 
-        $isCallbackInterface = $currentTransport instanceof InterfaceCallbackTransport || $currentTransport instanceof CallbackTransportInterface;
```

Comment:
If anyone else would be wondering if this can be removed, the answer is yes. See https://github.com/mautic/mautic/blob/916cb750f2e7896c735b231c98fd9a58c65cf0e5/app/bundles/EmailBundle/Swiftmailer/Transport/InterfaceCallbackTransport.php